RecoLocalCalo/EcalRecProducers/plugins/alpaka/EcalUncalibRecHitMultiFitAlgoPortable.h

```diff
@@ -0,0 +1,30 @@
+#ifndef RecoLocalCalo_EcalRecProducers_plugins_alpaka_EcalUncalibRecHitMultiFitAlgoPortable_h
+#define RecoLocalCalo_EcalRecProducers_plugins_alpaka_EcalUncalibRecHitMultiFitAlgoPortable_h
+
+#include <vector>
+
+#include "CondFormats/EcalObjects/interface/alpaka/EcalMultifitConditionsDevice.h"
+#include "CondFormats/EcalObjects/interface/alpaka/EcalMultifitParametersDevice.h"
+#include "DataFormats/EcalDigi/interface/alpaka/EcalDigiDeviceCollection.h"
+#include "DataFormats/EcalRecHit/interface/alpaka/EcalUncalibratedRecHitDeviceCollection.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/traits.h"
+#include "DeclsForKernels.h"
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE::ecal::multifit {
+
+  using InputProduct = EcalDigiDeviceCollection;
+  using OutputProduct = EcalUncalibratedRecHitDeviceCollection;
+
+  void entryPoint(Queue&,
+                  InputProduct const&,
+                  InputProduct const&,
+                  OutputProduct&,
+                  OutputProduct&,
+                  EcalMultifitConditionsDevice const&,
+                  EcalMultifitParametersDevice const&,
+                  ConfigurationParameters const&);
```

Comment:
Can you add the arguments' names ?
```suggestion
  void entryPoint(Queue& queue,
                  InputProduct const& digisDevEB,
                  InputProduct const& digisDevEE,
                  OutputProduct& uncalibRecHitsDevEB,
                  OutputProduct& uncalibRecHitsDevEE,
                  EcalMultifitConditionsDevice const& conditionsDev,
                  EcalMultifitParametersDevice const& paramsDev,
                  ConfigurationParameters const& configParams);
```

I would also use a more descriptive name than `entryPoint`, but that can be left for later (https://github.com/cms-sw/cmssw/issues/43551).